lib/node_modules/@stdlib/math/base/special/rising-factorial/test/test.native.js

```diff
@@ -0,0 +1,105 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var resolve = require( 'path' ).resolve;
+var tape = require( 'tape' );
+var tryRequire = require( '@stdlib/utils/try-require' );
+var abs = require( '@stdlib/math/base/special/abs' );
+var EPS = require( '@stdlib/constants/float64/eps' );
+
+
+// FIXTURES //
+
+var data = require( './fixtures/cpp/output.json' );
+
+
+// VARIABLES //
+
+var risingFactorial = tryRequire( resolve( __dirname, './../lib/native.js' ) );
+var opts = {
+	'skip': ( risingFactorial instanceof Error )
+};
+
+
+// TESTS //
+
+tape( 'main export is a function', opts, function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof risingFactorial, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function returns `1` if provided `n = 0` and a nonnegative `x`', opts, function test( t ) {
+	var val = risingFactorial( 2.0, 0 );
+	t.equal( val, 1.0, 'returns expected value' );
```

Comment:
```suggestion
	t.strictEqual( val, 1.0, 'returns expected value' );
```

Applies here and elsewhere.